src/format.py

```diff
@@ -108,6 +110,39 @@ def getScreenIndex(self):
     def getPath(self):
         return self.path
 
+    def getSizeInBytes(self):
+        bashCommand = "ls -lh " + self.path
+        output = subprocess.check_output(bashCommand.split())
+        size = output.split()[4]
+        return 'size: ' + str(size)
+
+    def getLengthInLines(self):
+        bashCommand = "wc -l " + self.path
+        output = subprocess.check_output(bashCommand.split())
+        return 'length: ' + str(output.strip().split()[0]) + ' lines'
+
+    def getTimeLastAccessed(self):
+        timeAccessed = time.strftime('%m/%d/%Y %H:%M:%S', time.localtime(os.stat(self.path).st_atime))
+        return 'last accessed: ' + timeAccessed
+
+    def getTimeLastModified(self):
+        timeModified = time.strftime('%m/%d/%Y %H:%M:%S', time.localtime(os.stat(self.path).st_mtime))
+        return 'last modified: ' + timeModified
+
+    def getOwnerUser(self):
+        bashCommand = "ls -ld " + self.path
+        output = subprocess.check_output(bashCommand.split())
+        userOwnerName = output.split()[2]
```

Comment:
i guess the format here is always consistent so we are safe to index with confident